packages/core/src/generators/qwik/jsx.ts

```diff
@@ -165,22 +165,25 @@ function rewriteHandlers(
   const outBindings: { [key: string]: { code: string; arguments?: string[] } } = {};
   for (let key in bindings) {
     if (Object.prototype.hasOwnProperty.call(bindings, key)) {
-      let { code: binding } = bindings[key]!;
+      let bindingExpr: string | undefined =
+        typeof bindings[key] == 'object' ? bindings[key]?.code : (bindings[key] as any);
```

Comment:
Weird: why do we need this change? `bindings[key]` should _always_ be a `{ code: string; arguments?: string[] } | undefined`. 

To avoid the `!` assertion, you can do this instead:

```suggestion
      let bindingExpr: string | undefined = bindings?.[key]?.code
```

Reply:
> Weird: why do we need this change? bindings[key] should always be a { code: string; arguments?: string[] } | undefined.

Yes, it should be, but it is not in some cases. 🤷 

Will incorporate your suggestion.

Reply:
Actually, your suggestion changes the meaning of the code so I don't think I can take it as is.

Reply:
> Yes, it should be, but it is not in some cases. 🤷

I guess that's what I'm trying to dig into: can you give an example of when this is happening? That should be impossible with the current version of Mitosis JSX parser.

Only way you're encountering this is if you're reading some old Mitosis component JSON that you've hardcoded/stored somewhere. The old format was `bindings: string`, but we've completely replaced that everywhere. 

My guess is you have some test specs that are stuck with the old format?

Reply:
OK switched to your case, and all tests pass. I think I was just being paranoid.